### PR TITLE
[16.0] [MIG] knowledge and rename it as document_knowledge

### PR DESCRIPTION
**Pull request by marielejeune**

### CODE REVIEW

**Comment by marielejeune** [comment]
@pedrobaeza I tried to re-create a PR here

**Comment by pedrobaeza** [comment]
It worked!

Selección_002

File: document_knowledge/i18n/knowledge.pot [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* knowledge
```
Comment (MiquelRForgeFlow) [inline document_knowledge/i18n/knowledge.pot:3]:
```suggestion
# 	* document_knowledge
```
and also rename the file to document_knowledge.pot

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by apineux — APPROVED** [review]

**Comment by pedrobaeza** [comment]
As the bot will do the POT/PO adaptations without losing the translations, let's not block more time this:

/ocabot migration knowledge
/ocabot merge nobump

**Pull request merged**